azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -943,30 +943,6 @@ private void ajaxFetchExecutableFlow(final HttpServletRequest req,
     ret.putAll(flowObj);
   }
 
-  private void ajaxAttemptExecuteFlow(final HttpServletRequest req,
```

Comment:
This method duplicated work done in ajaxExecuteFlow(..) and is not used anywhere else.